src/include/planner/operator/scan/logical_scan_internal_id.h

```diff
@@ -29,5 +29,29 @@ class LogicalScanInternalID : public LogicalOperator {
     std::vector<common::table_id_t> tableIDs;
 };
 
+class LogicalFillTableID : public LogicalOperator {
+public:
+    LogicalFillTableID(std::shared_ptr<binder::Expression> internalID, common::table_id_t tableID,
```

Comment:
Let's open an issue on this about removing this by implementing materialized `INTERNAL_ID`.